wiki/Beatmapping/Beats_per_minute/en.md

```diff
@@ -0,0 +1,19 @@
+---
+tags:
+  - BPM
+  - Tempo
+---
+
+# Beats per minute
+
+**Beats Per Minute**, abbreviated as *BPM*, refers to the number of whole beats in a minute. BPM is used to measure the [tempo](wikipedia.org/wiki/Tempo) of a song. This is needed to line up the beats of a song with the [hit objects](/wiki/Hit_Objects) of a [beatmap] successfully. It can be changed in the [beatmap editor](/wiki/Beatmap_Editor) under the `Timing` tab, or by using the shortcut (`Ctrl` + `P`).
+
+Most modern songs use a single BPM which requires a single timing point. More complex songs may require multiple timing points and timing resets to adjust succesfully. Timing points are shown as red lines in the timeline at the bottom of the editor.
```

Comment:
```suggestion
Most modern songs use a single BPM which requires a single timing point. More complex songs may require multiple timing points and timing resets to adjust successfully. Timing points are shown as red lines in the timeline at the bottom of the editor.
```
suggestion fixes a typo, but can you elaborate or explain what you mean by `timing resets to adjust successfully`?

Reply:
should be "to be timed correctly". timing resets and timing points should each have their own articles tbh.